priv/repo/migrations/20200710123927_create_participants.exs

```diff
@@ -0,0 +1,20 @@
+defmodule LiquidVoting.Repo.Migrations.CreateParticipants do
+  use Ecto.Migration
+
+  def change do
+    create table(:participants, primary_key: false) do
```

Comment:
I'm wondering if we could configure the DB to use primary keys in the `config.exs` as was shown here: https://annkissam.com/elixir/alembic/posts/2019/01/04/ecto-and-binary-ids.html?

```elixir
config :my_app, MyApp.Repo,
  migration_primary_key: [name: :id, type: :uuid]
```

Reply:
OK, I'll try this.